plugins/workspace-backpack/src/backpack.js

```diff
@@ -457,16 +461,16 @@ export class Backpack extends Blockly.DragTarget {
    *     provided block.
    */
   containsBlock(block) {
-    const cleanedBlockXml = this.blockToCleanXmlString_(block);
-    return this.contents_.indexOf(cleanedBlockXml) !== -1;
+    const blockJSONString = this.blockToJSONString_(block);
+    return this.contents_.indexOf(blockJSONString) !== -1;
```

Comment:
Can you remove the unnecessary variable assignments (here and below)? I know they predated your change :P but would be nice to clean them up while you're in the area!